Userland/Libraries/LibWeb/HTML/CanvasRenderingContext2D.cpp

```diff
@@ -195,6 +195,7 @@ void CanvasRenderingContext2D::stroke()
         return;
 
     painter->stroke_path(m_path, m_stroke_style, m_line_width);
+    did_draw(m_path.bounding_box());
```

Comment:
Nitpick about commit message: the layout (position and geometry of each `Web::Layout::Node`) is not being invalidated here, `did_draw()` only calls `set_needs_display()` on the canvas element's layout node (a `Web::Layout::CanvasBox`), meaning that it needs to be repainted.

Reply:
my bad, fixed!